server/dive_server/serializers/kwcoco.py

```diff
@@ -0,0 +1,228 @@
+"""
+KWCOCO JSON format deserializer
+"""
+import functools
+import json
+from typing import Any, Dict, List, Tuple
+
+from dive_server.serializers import viame
+from dive_utils import strNumericCompare
+from dive_utils.models import Attribute, CocoMetadata, Feature, Track
+
+
+def is_coco_json(coco: Dict[str, Any]):
+    keys = ['categories', 'keypoint_categories', 'images', 'videos', 'annotations']
+    return any(key in coco for key in keys)
+
+
+def is_coco_string(coco: str):
+    data = json.loads(coco)
+    keys = ['categories', 'keypoint_categories', 'images', 'videos', 'annotations']
+    return any(key in data for key in keys)
```

Comment:
You're right. I wrote the `is_coco_string` function originally when I was reading the Girder File as a string (the same way the viame csv is parsed). However, I realized later that it's possible to get a file handler for the Girder File and wrote `is_coco_json`. Since `is_coco_string` is no longer being used, I think I should just get rid of the extraneous function.